README.md

```diff
@@ -1,3 +1,157 @@
 # redux-entitize
 
-Automated [redux](https://github.com/reactjs/redux)-state management for all your entities.
\ No newline at end of file
+Automated [redux](https://github.com/reactjs/redux)-state management for all your entities.
+
+`redux-entitize` maintains normalized entities in the state and provides:
+
+- an entities reducer
+- entity update / delete action creators
+- entity selectors
+
+## Getting started
+
+### Install
+
+```bash
+npm install --save redux-entitize
+```
+
+or
+
+```bash
+yarn add redux-entitize
+```
+
+### Usage Example
+
+`redux-entitize` provides a creator function for a single entities reducer handles the update and delete operations on your entities.
```

Comment:
maybe "provides a factory function to create a reducer. This is needed because the reducer needs to know your `normalizr`-schemas in order to normalize your entities before putting them into the state.